src/index.js

```diff
@@ -294,154 +326,145 @@ const topVrPresentState = {
   vrSystem: null,
   vrCompositor: null,
   hasPose: false,
+  mesher: null,
+  planesTracker: null,
 };
 
-const _startTopRenderLoop = () => {
-  const timestamps = {
-    frames: 0,
-    last: Date.now(),
-    idle: 0,
-    wait: 0,
-    events: 0,
-    media: 0,
-    user: 0,
-    submit: 0,
-    total: 0,
-  };
-  const TIMESTAMP_FRAMES = 100;
+const requestPresent = async () => {
+  const hmdType = getHMDType();
+  console.log('request present', hmdType); // XXX
```

Comment:
I've noticed the `// XXX` comment in different areas, what does it signify?

Reply:
I use it as a todo note, often a log to consider removing.